### ISSUE TITLE
Highlight color causes icons to break with Catppuccin Mocha

### ISSUE THREAD

**Issue by sbeve72** [catppuccin/obsidian#39]
As you can see, with the spaces plugin, sometimes highlight color and text color/icon color has really low contrast. Is there something that can be done to fix this?

https://user-images.githubusercontent.com/65838896/232799841-573e6c49-f4ba-4c6c-8a44-6067321cb76b.mp4

**Comment by mbeckrich** [comment]
Yep, let me download the plugin and get the theme working with it. I’ll try to have it fixed today.

**Comment by sbeve72** [comment]
Thanks a lot!

**Comment by mbeckrich** [comment]
I'm working on this right now -- hadn't used the plugin before, so I've a couple questions if you don't mind.

1) On loading the plugin, I notice that there are duplicate icons in the left sidebar's file tree. I don't see that in your video, so is there an option within MAKE.md to fix this or did you fix it manually yourself? If you had to do it yourself, let me know and I will see what I can do to hide the theme icons when MAKE.md is active.

2) It looks like there are a lot of UI elements added with MAKE.md and the Catppuccin theme doesn't style them very well right now. Are there settings within MAKE.md that can fix this or does the user have to fix it themselves with css? Either way, I'll try to style the plugin elements by default, but was just wondering if there's an option within the plugin that people use to change things on their own, too.

**Comment by sbeve72** [comment]
1. Yes, you can disable stickers in the settings. They're not really double icons more like, the new ones are placeholders for emojis.
2. It looks like the only way to fix them is through CSS since I don't see any option within Make.MD's settings.

One question, when you're done making these changes, can you also make it so that the Page titles (the name of the .md file in the note's editor view) is the same color H1 titles?

**Comment by mbeckrich** [comment]
Gotcha, thanks.

And yes, I can do that. Are you using the Style Settings plugin? I might add something there for people to adjust the page title color more broadly, because I could see some people preferring to keep the page title a separate color from the header colors. Or maybe a heavier weight to the page title? Something like that.

**Comment by sbeve72** [comment]
Yes, I am using the style settings plugin. It would be nice if we could just change the color of the Page title to one of the colors from the list that appears when we try to change heading colors.

**Comment by mbeckrich** [comment]
Awesome, I’ll make it happen. Appreciate your thoughts on all this. :)

**Comment by mbeckrich** [comment]
I've just pushed the update, so once you're able to take a look, let me know how it feels. Hopefully contrast is improved. I see that the plugin has a little command palette of its own, haven't been able to get to that completely but I started on it.

**Comment by mbeckrich** [comment]
Thanks, super helpful. I will look into this more later this evening and try to have that stuff fixed soon.

**Comment by sbeve72** [comment]
Thanks, really appreciate that.

**Comment by mbeckrich** [comment]
Okay, let me know how it feels now. I've also remodeled the buttons just a bit to get them to clash less with the file type tags. I notice the file names jump around when hovered, but that seems to happen when using just the default theme, too, so I assume it is a feature that can be toggled w/in the plugin settings?

**Comment by sbeve72** [comment]
> I notice the file names jump around when hovered, but that seems to happen when using just the default theme, too, so I assume it is a feature that can be toggled w/in the plugin settings?
I am not sure about that but the theme looks perfect now. No issues anymore only small aesthetic irks that I don't bother me much.

**Comment by mbeckrich** [comment]
If you have specific feedback on aesthetic choices, feel free to open a new issue and we can discuss. Or you can submit a PR we can review together — I’ve made a doc outlining contributions in the repo. 

I’m not averse to making changes provided they fit within the broader rules for the theme as they were explained to me: basically, nothing opinionated and stick to the palette.
